charts/schema-loading/templates/batchjob-schema-loading.yaml

```diff
@@ -76,5 +89,20 @@ spec:
               key: db-password
         - name: SCHEMA_TYPE
           value: "{{ .Values.schemaLoading.schemaType }}"
+        {{- if .Values.schemaLoading.secretName }}
+        envFrom:
+        - secretRef:
+            name: "{{ .Values.schemaLoading.secretName }}"
+        {{- end }}
+      {{- if .Values.schemaLoading.databaseProperties }}
+        volumeMounts:
+          - name: scalardb-database-properties-volume
+            mountPath: /scalardl-schema-loader/database.properties.tmpl
+            subPath: database.properties.tmpl
+      volumes:
+        - name: scalardb-database-properties-volume
+          configMap:
+            name: {{ include "schema-loading.fullname" . }}-database-properties
+      {{- end }}
```

Comment:
The value of `schemaLoading.databaseProperties` is translated to a `ConfigMap`.
This chart mounts it in the container in the same way as the Scalar DB/DL chart.